### DISCUSSION TITLE
How to adjust the text size of project area?

### DISCUSSION THREAD

**Discussion by oceanfish567** [apache/netbeans#8091] [General]
### Please see the picture description, Thank you.

![PixPin_2024-12-25_06-32-49](https://github.com/user-attachments/assets/4bb5373b-e018-44e4-b1b7-9dfb7ee04b81)

**Comment by matthiasblaesing** [comment]
Maybe it helps to know, that the base font size of the UI can be changed with command line parameter "fontsize". Have a look here:

```
matthias@enterprise:~/src/netbeans/nbbuild/netbeans$ ./bin/netbeans --help 2>&1 | grep fontsize
  --fontsize <size>     set the base font size of the user interface, in points
matthias@enterprise:~/src/netbeans/nbbuild/netbeans$
```

The font size of the treeview should be derived from that.

**Comment by hantsy** [comment]
Open `etc/netbeans.conf`, add `--fontsize 16` to the `netbeans_default_options`.

**Comment by oceanfish567** [comment]
Thank you everyone, I finally found a better adjustment method that will not interfere with other UI styles.

Use "FlatLaf" custom mode and add a piece of setting code
`Tree.font=16`
That's it

![圖片](https://github.com/user-attachments/assets/c92bc358-104d-4037-9efe-9ea5af6aff7a)

**Reply by Chris2011** [reply]
Thx for letting us now :)
